src/routes/index.tsx

```diff
@@ -10,38 +10,36 @@ import SpotsPage from "pages/spots";
 
 const isMaintenanceMode = MAINTENANCE_MODE === 'true' ? true : false;
 
-const DEFAULT_MESSAGE = {message: "Cannot load page. Please try again."}
+function RootBoundary() {
+  const error = useRouteError();
+  return <ErrorPage error={error} />;
+}
```

Comment:
Since the errors 'bubble up' in react router dom and we aren't doing anything special for individual routes, we can just show the error message from the exception or display a default (done in the ErrorPage component).